backend/de.metas.procurement.base/src/main/java/de/metas/procurement/base/impl/WebuiPush.java

```diff
@@ -103,11 +103,6 @@ public void pushBPartnerAndUsers(final I_C_BPartner bpartner)
 // @formatter:on
 
 		final SyncBPartner syncBPartner = syncFactory.createSyncBPartnerWithoutContracts(bpartner);
-		if (syncBPartner == null)
-		{
-			return;
-		}
-
 		final PutBPartnersRequest syncBPartnersRequest = PutBPartnersRequest.of(syncBPartner);
 		agent.syncBPartners(syncBPartnersRequest);
```

Comment:
we should record this in the export audit log, and make sure to have a reference between it and the RabbitMQ_Message_Audit record